propka/protonate.py

```diff
@@ -137,6 +149,10 @@ def set_number_of_protons_to_add(self, atom):
         _LOGGER.debug('Setting number of protons to add for %s', atom)
         atom.number_of_protons_to_add = 8
         _LOGGER.debug("                     8")
+        if atom.element not in self.valence_electrons:
+            _LOGGER.warning(f'Unknown valence electron \
+                            for element {atom.element}')
```

Comment:
This will put 28 literal spaces in your string and print something like:
```
Not found valence for                              element Foo, use 4
```
I suggest to simply keep it on one line.